### PR TITLE
Set max width to add permissions filter

### PR DESCRIPTION
**Pull request by PanSpagetka**
Filter select was too wide and pushed pagination on next row.

Before
Screenshot from 2020-09-25 10-08-24

After
Screenshot from 2020-09-25 10-07-52

@mmenestr

### CODE REVIEW

File: src/smart-components/role/add-role-new/add-role-wizard.scss [hunk 1]
```diff
@@ -40,4 +40,20 @@
...
 
-
+.ins-c-rbac-permissions-table {
+    .ins-c-primary-toolbar {
+        .pf-c-toolbar__content {
+            .pf-c-toolbar__content-section {
+                .pf-m-filter-group {
+                    .ins-c-primary-toolbar__filter {
+                        .ins-c-conditional-filter {
+                            .pf-c-select {
```
Comment (karelhala) [inline src/smart-components/role/add-role-new/add-role-wizard.scss:50]: Phew, this is really complex. Isn't it possible to simplify it a bit?

Comment (PanSpagetka) [reply src/smart-components/role/add-role-new/add-role-wizard.scss]: I didn't want to change the Toolbar component, so there is quite complex selection, but I think I can simplify it a bit.

**Comment by mmenestr** [comment]
Perfect, thank you!!

File: src/smart-components/role/add-role-new/add-role-wizard.scss [hunk 2]
```diff
@@ -42,3 +42,15 @@
         }
     }
 }
+
+.ins-c-rbac-permissions-table {
+    .ins-c-primary-toolbar {
+        .pf-c-toolbar__content {
+            .pf-c-toolbar__content-section {
+                .pf-m-filter-group {
+                    max-width: 35em;
```
Comment (Hyperkid123) [inline src/smart-components/role/add-role-new/add-role-wizard.scss:51]: How responsive is this? I think that using a % value might be a bit safer.

Comment (PanSpagetka) [reply src/smart-components/role/add-role-new/add-role-wizard.scss]: I have experiment a bit with media queries and it seems to be working well now. Even though it is a bit complex. 
Unfortunately, github does not support `.webm` files https://webm.red/view/YtI4.webm
(I can't make gifs)

Comment (Hyperkid123) [reply src/smart-components/role/add-role-new/add-role-wizard.scss]: Yeah that looks good. Can you fix up travis?

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**